azuredeploy.json

```diff
@@ -271,45 +271,49 @@
             },
             "type": "string"
         },
-        "skuName": {
+        "mssqlDbEdition": {
             "allowedValues": [
-                "PGSQLB50",
-                "PGSQLB100",
-                "PGSQLS100",
-                "PGSQLS200",
-                "PGSQLS400",
-                "PGSQLS800",
-                "MYSQLB50",
-                "MYSQLB100",
-                "MYSQLS100",
-                "MYSQLS200",
-                "MYSQLS400",
-                "MYSQLS800"
+                "Basic",
+                "Standard"
             ],
-            "defaultValue": "MYSQLS100",
+            "defaultValue": "Standard",
             "metadata": {
-                "description": "MySql/Postgresql sku name"
+                "description": "MS SQL DB edition. There are a lot more than Basic & Standard (to be added later). See https://docs.microsoft.com/en-us/rest/api/sql/databases/createorupdate#DatabaseEdition"
```

Comment:
Ditto the comment above, this is not appropriate for a help documentation string.

Reply:
Ditto.